pkgs/by-name/ze/zerotierone-desktop-ui/package.nix

```
@@ -0,0 +1,95 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, gnumake
+, gcc
+, zlib
+, freetype
+, fontconfig
+, gtk3
+, libappindicator
+}:
+
+
+let
+  pname = "zerotierone-desktop-ui";
+  version = "2024.9.0"; # Infrequent updates and does not seem to be released often
+
+  src = fetchFromGitHub {
+    owner = "zerotier";
+    repo = "DesktopUI";
+    rev = "452eca3270f84ef4ac81c955f4777c8f1a82b988";
+    hash = "sha256-1gYF9v7sxCO7+pBRBSCYSRaoHEW3Xvx7nh28TsERZy8=";
+  };
+in
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage {
```
This doesn't need to be recursive.

---

removed